libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.ts

```diff
@@ -311,6 +324,8 @@ export class SkyTextEditorComponent
   #_mergeFields: SkyTextEditorMergeField[] = [];
   #_menus = MENU_DEFAULTS;
   #_toolbarActions: SkyTextEditorToolbarActionType[] = TOOLBAR_ACTION_DEFAULTS;
+  #_linkWindowOptions: SkyTextEditorLinkWindowOptionsType[] =
```

Comment:
Since `LINK_WINDOW_OPTIONS_DEFAULT` already has a type specified, `#_linkWindowOptions` can infer its type from `LINK_WINDOW_OPTIONS_DEFAULT` and doesn't need it specified here.

```
#_linkWindowOptions = LINK_WINDOW_OPTIONS_DEFAULT;
```

Reply:
Done!